one.mixin.messenger.desktop.yml

```diff
@@ -0,0 +1,51 @@
+id: one.mixin.messenger.desktop
+runtime: org.gnome.Platform
+runtime-version: '42'
+sdk: org.gnome.Sdk
+command: mixin_desktop
+rename-desktop-file: mixin_desktop.desktop
+rename-icon: mixin_desktop
+rename-appdata-file: mixin_desktop.metainfo.xml
+finish-args:
+  - --share=ipc
+  - --socket=fallback-x11
+  - --socket=wayland
+  - --device=dri
+  - --socket=pulseaudio
+  - --share=network
+  - --filesystem=home
```

Comment:
this is a broad permission for a messenger app.

Reply:
Does it measings that we can't use these permissions?